app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  scope :updates_likes_counter, lambda { |id_value|
+                                  Post.find(id_value).update(likes_counter: Like.where(post_id: id_value).count(:user_id))
+                                }
+end
```

Comment:
- [ ] [OPTIONAL] You did a great job here, scope are good because they are easy to be used, but did you know that we can solve the same problem using a short option? Take a look below, you are free to make the change
```ruby
def update_likes_counter
  author.increment!(:like_counter)
end
```

Reply:
Thanks, done!